client/src/modules/reports/deprecated-modals/accounts_chart.modal.html

```diff
@@ -5,15 +5,15 @@
   </ol>
 </div>
 
-<form name="ConfigForm" bh-submit="ReportConfigCtrl.generate()" novalidate>
+<form name="ConfigForm" bh-submit="ReportConfigCtrl.generate()" bh-form-defaults novalidate>
```

Comment:
`bhFormDefaults` has been deprecated in https://github.com/IMA-WorldHealth/bhima-2.X/issues/1447 and completely removed in https://github.com/IMA-WorldHealth/bhima-2.X/pull/1755.  It should be removed from these modules.